libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html

```diff
@@ -1,29 +1,25 @@
-<gn-ui-content-ghost
+<div
+  *ngIf="imageUrl"
```

Comment:
Looks like the "content-ghost" for the thumbnail is gone, could you please bring it back and see if your changes still work? thanks :)

Reply:
I removed that because it was creating a weird display of the thumbnail, but if it's important I can look into having it back.

Reply:
hmm, maybe the content ghost doesn't work well on a colored background? what do you mean by "weird display"? The content ghost is important to avoid layout shifts when the data loading is finished

Reply:
Basically, it's like the image is hidden by a mask and there is a "loading animation"... Also, the console shows no error, so I'm a bit at lost on what is wrong here.

But maybe I'm missing something, like a parameter ? Here I have added back the "content-ghost" and its class `ghostClass="h-36 mb-3"`. 

![image](https://github.com/geonetwork/geonetwork-ui/assets/132347903/20f3885d-08bf-48d1-b2e8-163fff5042a5)